packages/coil-extension/src/background/services/Stream.ts

```diff
@@ -129,38 +124,55 @@ export class Stream extends EventEmitter {
   }
 
   async start() {
-    if (!this._active) {
-      this._packetNumber = 0
-      this._active = true
-
-      // Hack for for issue #144
-      // Let pause() stream when tab is backgrounded have a chance to
-      // to work to avoid wasted refreshBtpToken/SPSP queries
-      await timeout(1)
-      if (!this._active) return
-
-      // reset our timer when we start streaming.
-      this._bandwidth.reset()
-
-      while (this._active) {
-        try {
-          await this._stream()
+    if (this._active) return
+    this._active = true
+    if (this._looping) return
+    this._looping = true
```

Comment:
I think I understand this. You could stop then restart so fast that multiple loops are in action at once right ?

Reply:
Yes, that's what I'm trying to avoid.